### DISCUSSION TITLE
False Positive Malware?

### DISCUSSION THREAD

**Discussion by Dsnherdude066** [AntiMicroX/antimicrox#1049] [General] [answered]
ive had antimicro from the Microsoft store for a while now and wanted to "upgrade" recently but because im a paranoid mf, so i scanned the file on VT(virustotal) and it comes up with walware, i made a post on the antivirus reddit and was told to contact the creator(s) and just wanted to know why any false positives would come up, Link to reddit post: https://www.reddit.com/r/antivirus/comments/1ftann7/just_need_someone_to_confirm_if_these_are_safe/

**Comment by pktiuk** [comment, marked as answer]
AntiMicroX is not officially distributed using Microsoft Store.     
AntiMicro (without X) is a long-abandoned project, not developed anymore.

Anyway, **I do not recommend** downloading it from Microsoft store. Maybe you should report it to Microsoft.

The only official package for Windows can be downloaded here: https://github.com/AntiMicroX/antimicrox/releases/latest 
And that one is fine, it is automatically built from open source code. If you want, you can even try to build it by yourself.

**Reply by Dsnherdude066** [reply]
i scanned AntiMicroX not antimicro, sorry for that confusion
